### DISCUSSION TITLE
Display Only Today's and Overdue Tasks

### DISCUSSION THREAD

**Discussion by purpleturtle1988** [fixtse/o365-card#4] [Q&A] [answered]
Hello fixtse,

I'm currently using your task management card and it's been incredibly helpful. However, I have a feature request that could enhance its functionality for users like me. Would it be possible to implement a feature where the card only displays tasks that are due today, along with any overdue tasks? This would greatly help in focusing on immediate priorities and managing overdue items more effectively.

Looking forward to your thoughts on this.

Best regards

**Comment by fixtse** [comment, marked as answer]
Hi, sure, it seems like an efficient way of managing your tasks, Let me see if I can add the option in the next beta release (I'm working on adding support for the new task sensor that home assistant just introduced, but I'll have to wait until the o365 integration releases a new version to release it as stable)
